core/NameRef.h

```diff
@@ -74,9 +104,11 @@ class NameRef final : private DebugOnlyCheck<NameRefDebugCheck> {
         return _id;
     }
 
-    NameData data(GlobalState &gs) const;
+    NameKind kind(const GlobalState &gs) const;
```

Comment:
In the next change (breaking up the name table), `kind` will be stored in the `NameRef` directly and will not require a GlobalState argument.